### PR TITLE
Update entrypoint and readme to improve quick-start.

### PR DESCRIPTION
**Pull request by boweiliu**
Resolves https://github.com/factoriotools/factorio-docker/issues/450.

Should make the quick start copy-paste snippet "just work".

### CODE REVIEW

**Comment by boweiliu** [comment]
Hi @Fank any interest in getting this merged?

File: README.md [hunk 1]
```diff
@@ -53,6 +54,7 @@ sudo docker run -d \
 For those new to Docker, here is an explanation of the options:
 
 * `-d` - Run as a daemon ("detached").
+* `-it` - Interactive mode.
```
Comment (SuperSandro2000) [inline README.md:57]: That is so incorrect. -i is interactive and -t allocates a tty.

File: README.md [hunk 2]
```diff
@@ -75,22 +77,49 @@ docker stop factorio
...
+2022-10-23 00:26:03 [CHAT] <server>: hello everyone!
 
-```shell
-docker run -d -it  \
-      --name factorio \
-      factoriotools/factorio
-docker attach factorio
 ```
 
+To detach and return to outside docker, use Ctrl-P + CTRL-Q.
```
Comment (SuperSandro2000) [inline README.md:121]: You are sure this is not shell depended?

File: docker/files/docker-entrypoint.sh [hunk 3]
```diff
@@ -21,7 +21,8 @@ fi
 
 if [[ ! -f $CONFIG/server-settings.json ]]; then
   # Copy default settings if server-settings.json doesn't exist
-  cp /opt/factorio/data/server-settings.example.json "$CONFIG/server-settings.json"
+  # edit the default visibility setting from public: true to public: false
+  cat /opt/factorio/data/server-settings.example.json | sed 's/    "public": true/    "public": false/'  > "$CONFIG/server-settings.json"
```
Comment (SuperSandro2000) [inline docker/files/docker-entrypoint.sh:25]: please use jq for this

**Pull request closed without merging**